LibreNMS/Alert/Transport/Alertmanager.php

```diff
@@ -0,0 +1,110 @@
+<?php
+/* Copyright (C) 2019 LibreNMS
+ * This program is free software: you can redistribute it and/or modify
+ * it under the terms of the GNU General Public License as published by
+ * the Free Software Foundation, either version 3 of the License, or
+ * (at your option) any later version.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.See the
+ * GNU General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public License
+ * along with this program.  If not, see <http://www.gnu.org/licenses/>. */
+
+/**
+ * API Transport
+ * @copyright 2019 LibreNMS
+ * @license GPL
+ * @package LibreNMS
+ * @subpackage Alerts
+ */
+namespace LibreNMS\Alert\Transport;
+
+use LibreNMS\Alert\Transport;
+use LibreNMS\Config;
+
+class Alertmanager extends Transport
+{
+    public function deliverAlert($obj, $opts)
+    {
+        $alertmanager_opts['url'] = $this->config['alertmanager-url'];
```

Comment:
Scrutinizer: ``$alertmanager_opts`` was never initialized. Although not strictly required by PHP, it is generally a good practice to add ``$alertmanager_opts = array();`` before regardless.